spinalcordtoolbox/reports/qc.py

```diff
@@ -155,6 +160,14 @@ def wrapped_f(sct_slice, *args):
 
             img, mask = func(sct_slice, *args)
 
+            if self._stretch_contrast:
+                def equalized(a):
+                    _min, _max = a.min(), a.max()
```

Comment:
Not sure what you mean. `_min, _max` are local variables (so kind of internal use...) and are different than the function names, which are not used anyway. Not sure where the PEP8 conflict is also.

Reply:
There are two python built-ins functions called `min()` and `max()`. The recommended pattern in PEP8 to avoid conflicting with these built-in names when the variable name is the same, is to use an underscore at the end of the variable, such as: `min_` and `max_`. The underscore in the beginning of it is reserved for functions such as `def _internal_method(): # (...)` or class attributes.

Reply:
I have vaguely heard about these built-in functions and PEP8 ;)
If you can point me to that particular section, I'll take a look and change the offending lines.

Reply:
![image](https://user-images.githubusercontent.com/412328/38067778-0054ea68-32dc-11e8-9bc9-d8d0f3ea56ff.png)

Reply:
I'm not convinced that this is suitable context for this case, but I changed the code as per your suggestion.